contents/handbook/company/management.md

```diff
@@ -0,0 +1,57 @@
+1. Making sure their direct reports are happy and productive
+1. Setting the right context for direct reports to do their job
+
+That's it.
+
+A manager at PostHog is _not_ responsible for:
+1. Setting compensation (we have transparent compensation)
+1. Setting tasks for their direct reports
+1. Creating a career path (career paths should be transparent and documented, and for now centrally managed)
+1. "Approving," whether that's projects, expenses, days off or accounts (people should have admin access by default to most things)
+
+
+## What does setting context mean?
+
+At PostHog, we exclusively hire people that are the best in their field.
+That means managers won't need to spend time telling their direct reports what to do.
+
+However, for those people to make the best decisions, they need context. That context can be
+- what a customer said was or wasn't important to them
+- what the metrics are saying needs to be improved
+- what another team in the organisation is working on
+- what the overal goals are for PostHog
+
+The shift here, and the biggest difference between PostHog and other places, is that in the end it is up to the individual to make the decisions.
+All you can do as a manager is set context. From there, you'll have to trust that we've made the right hiring decisions and that the individual is able to execute on that. If they can't, we have a [generous severance policy](/handbook/people/compensation#severance).
+
+Decisions aren't just about buying a piece of software or choosing a color for a button. It's also about what to work on, what to invest time in, or where to take entire parts of our product.
+Again, we've hired the best people and have high talent density, so we trust everyone to make these kinds of decisions.
+
+As a manager, it's tempting to see yourself as the sole owner of all the information, and give it out sparingly.
+People will come to you often with questions (because they don't have the context) and when they do you'll get more validation that holding all the context yourself makes you an Important Person.
+What managers should really aim for at PostHog is to make themselves obsolete. Share as much context as possible, preferably in written form. That way everyone will be able to do their best work.
+
+## Part-time managers
+
+Because of the relatively short list of tasks that managers have, management at PostHog is a part-time job.
+That means everyone, from CEO and CTO on up, still spend the majority of their time on practicing what they do best (which likely isn't management!).
+
+As an engineer, you wouldn't respect the opinion of someone who can't code on a coding question.
+As a designer, you really want your manager to have an eye for design.
+That's why it's important for managers to keep practicing their craft.
+
+## Hierarchy
+
+Another benefit of managers having a short task list is that they are able to have more direct reports, creating a flatter structure.
```

Comment:
Relates to @macobo's point above about relationship between manager and managee. I'm also wondering how homogenous we are planning this structure to be. 

As much as we're splitting into a _design_ team and an _engineering_ team, I could equally see how the team would need to be split into small groups with mixed skills such as a team of 1 designer and 3 engineers where 1 of these 4 people would be the "manager". This would arguably promote flatter structure because if once our team grows and we have double digit designers and engineers, if teams aren't cross functional the back and forth could be cumbersome.

Reply:
100% should not end up creating silos of just design/engineering/product/growth, otherwise we'll have a hard time implementing anything effectively.

Reply:
I think people can form teams under one manager (as is happening now while I still manage all engineers).

Yes agree on not creating silos. I think this raises another interesting question, if you have a team with designers and engineers does the manager need to be both good at design and engineering (my gut feel is yes though that's a very tough ask, realistically they need to have _some_ sense of design).

You could also end up with teams forming but people within them having different managers. I think this is a terrible idea as again it'll create more people involved in decisions, not fewer. Partly this is a bridge to cross when we get there.

Reply:
I think the _critical_ thing here is that task setting happens in a centralized, transparent way.

Very specifically, a manager should _not_ set tasks/priorities in the 1:1 if that person's work involves anyone else in any way, which is probably 99% of our work.

I think then it's probably a case of relying on people to just say who they need to work with. Ie I'd expect 

The alternative is to try to create managers who can look after ie designers/engineers/product people/marketing people, for example, which isn't realistic for the craft-practising reason above.

If we had that - managers that could literally do every role reasonaby well, I suspect we'd end up with then silos between cross functional teams, which will lead to things like stupid architecture or duplicated work. I think this is why in the task setting sessions we should be coordinating centrally who works with whom. This may mean tagging people outside of the meeting in before/after.